RESOURCES/FEATURE_FLAGS.md

```diff
@@ -52,7 +52,7 @@ These features are **finished** but currently being tested. They are usable, but
 - PLAYWRIGHT_REPORTS_AND_THUMBNAILS
 - RLS_IN_SQLLAB
 - SSH_TUNNELING [(docs)](https://superset.apache.org/docs/installation/setup-ssh-tunneling)
-- USE_ANALAGOUS_COLORS
+- USE_ANALOGOUS_COLORS
```

Comment:
oh this one is actually a breaking change, I would either not touch or you have to add a note in the `UPDATING.md` file.

My take on it would be to comment "# excuse the typo, but we'd rather not break backward compatibility"